go/systests/teams_test.go

```diff
@@ -1538,3 +1538,38 @@ func TestTeamBustResolverCacheOnSubteamRename(t *testing.T) {
 		require.Error(t, err)
 	}
 }
+
+func TestForceRepollState(t *testing.T) {
+	tt := newTeamTester(t)
+	defer tt.cleanup()
+
+	tt.addUser("onr")
+	tt.addUser("wtr")
+
+	_, err := tt.users[0].tc.G.API.Post(libkb.APIArg{
+		Endpoint:    "test/big_state_cutoff",
+		SessionType: libkb.APISessionTypeREQUIRED,
+		Args: libkb.HTTPArgs{
+			"cutoff": libkb.I{Val: 1},
```

Comment:
this would make concurrent tests real weird. Could pass team ID?

Reply:
only takes effect for the user posting!  so it should be fine for concurrent (if we don't use the user in other tests, which we don't!)